### PR TITLE
Make goat great again

### PR DESCRIPTION
**Pull request by hanfi**
le don de soi

### CODE REVIEW

File: assets/default.css [hunk 1]
```diff
@@ -69,3 +69,15 @@ header.header > .logo > img {
...
+  width:10%; 
+  position: absolute; 
+  top: 0; 
+  right: 0; 
+  border: 0;
+}
+
+header.header > .donate > img:hover {
+  width:100%; 
+}
```
Comment (deuzu) [inline assets/default.css]: No newline at the end of the file

File: index.html [hunk 2]
```diff
@@ -46,8 +46,11 @@
       <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="https://browsehappy.com/">upgrade your browser</a> to improve your experience and security.</p>
     <![endif]-->
     <header class="header">
+        <a class="donate" href="http://association-chevre-savoie.fr/don-association/">
```
Comment (deuzu) [inline index.html]: No https? =(

**Review by deuzu — CHANGES_REQUESTED** [review]

File: assets/default.css (hunk 1 above)
Comment (hanfi) [reply assets/default.css]: pardon fixed

**Review by deuzu — APPROVED** [review]

File: assets/default.css (hunk 1 above)
Comment (vandamic) [reply assets/default.css]: T'aurais pu passer le logo en svg :D

**Comment by hanfi** [comment]
But not deployed 👎

**Pull request merged**